[2025-05-12T07:20:10.318+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-12T07:20:10.349+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monitor_PWU.check_if_retrain_needed manual__2025-05-12T07:20:08.296412+00:00 [queued]>
[2025-05-12T07:20:10.358+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monitor_PWU.check_if_retrain_needed manual__2025-05-12T07:20:08.296412+00:00 [queued]>
[2025-05-12T07:20:10.358+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-05-12T07:20:10.372+0000] {taskinstance.py:2327} INFO - Executing <Task(BranchPythonOperator): check_if_retrain_needed> on 2025-05-12 07:20:08.296412+00:00
[2025-05-12T07:20:10.378+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'monitor_PWU', 'check_if_retrain_needed', 'manual__2025-05-12T07:20:08.296412+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/monitor_PWU.py', '--cfg-path', '/tmp/tmpfdlix3ku']
[2025-05-12T07:20:10.380+0000] {standard_task_runner.py:91} INFO - Job 48: Subtask check_if_retrain_needed
[2025-05-12T07:20:10.384+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=236) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-12T07:20:10.386+0000] {standard_task_runner.py:63} INFO - Started process 237 to run task
[2025-05-12T07:20:10.424+0000] {task_command.py:426} INFO - Running <TaskInstance: monitor_PWU.check_if_retrain_needed manual__2025-05-12T07:20:08.296412+00:00 [running]> on host 61ae3f5ac700
[2025-05-12T07:20:10.509+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monitor_PWU' AIRFLOW_CTX_TASK_ID='check_if_retrain_needed' AIRFLOW_CTX_EXECUTION_DATE='2025-05-12T07:20:08.296412+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-12T07:20:08.296412+00:00'
[2025-05-12T07:20:10.510+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-12T07:20:10.525+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-05-12T07:20:10.534+0000] {python.py:237} INFO - Done. Returned value was: trigger_retrain
[2025-05-12T07:20:10.535+0000] {branch.py:36} INFO - Branch into trigger_retrain
[2025-05-12T07:20:10.536+0000] {skipmixin.py:178} INFO - Following branch trigger_retrain
[2025-05-12T07:20:10.548+0000] {skipmixin.py:246} INFO - Skipping tasks [('end', -1)]
[2025-05-12T07:20:10.581+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-12T07:20:10.599+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=monitor_PWU, task_id=check_if_retrain_needed, execution_date=20250512T072008, start_date=20250512T072010, end_date=20250512T072010
[2025-05-12T07:20:10.641+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-12T07:20:10.667+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-12T07:20:10.669+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
